openacademy/models/openacademy_session.py

```diff
@@ -1,13 +1,93 @@
 from odoo import models
 from odoo import fields
+from odoo import api
+from odoo import exceptions
+from datetime import timedelta
+import time
 
 
 class Session(models.Model):
     _name = 'openacademy.session'
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today)
+    datetime_test = fields.Datetime(default=fields.Datetime.now)
     duration = fields.Float(digits=(6, 2), help="Duration in days")
     seats = fields.Integer(string="Number of seats")
+    instructor_id = fields.Many2one('res.partner', string='Instructor',
```

Comment:
> If a function call, dictionary, list or tuple is broken into two lines, break it at the opening symbol.

```suggestion
    instructor_id = fields.Many2one(
        'res.partner', string='Instructor',
```

You could see it in the guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id32

Apply this change for all the fields with this please.

Reply:
Sorry I didn't see this comment